build.gradle

```diff
@@ -58,8 +48,6 @@ if (project.hasProperty('ossrhUsername') && project.hasProperty('ossrhPassword')
 allprojects {
   // Formatting tasks
   // ================
-  apply plugin: 'com.github.sherter.google-java-format'
```

Comment:
What is the impact of moving this from `allprojects` to `subprojects`?

Reply:
There should be no practical impact (we do not have sources in the top module and do not have intention to have them there). I don't really understand why, but I could not make old setup work with formatter 1.6. 

Even without 1.6 formatter the new way should be more appropriate, because the old way was trying to format everything everywhere, including the build artifacts folders, which blew up after bazel build was added.